tests/CMakeLists.txt

```diff
@@ -53,36 +34,33 @@ if(BUILD_WITH_CODE_COVERAGE)
     add_dependencies(coverage aktualizr)
 endif()
 
-add_custom_target(qa)
 
 # Export compile_commands.json for clang-check
 set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
 
 
-set(ALL_TEST_SRCS   tests/config_test.cc
-                    tests/events_test.cc
-                    tests/commands_test.cc
-                    test/httpsotaclient_test.cc
-                    test/httpclient_test.cc)
+
+set(ALL_TEST_SRCS tests/commands_test.cc
+                  tests/config_test.cc
+                  tests/dbusgateway_test.cc
+                  tests/events_test.cc
+                  tests/httpclient_test.cc
+                  tests/httpsotaclient_test.cc
+                  tests/rvisotaclient_test.cc
+                  tests/socketgateway_test.cc
+                  tests/swm_test.cc)
 
 if(CLANG_FORMAT)
-add_custom_target(format-tools
-    COMMAND ${CLANG_FORMAT} -i -style file ${CMAKE_CURRENT_SOURCE_DIR}/src/*.cc ${CMAKE_CURRENT_SOURCE_DIR}/src/*.h ${CMAKE_CURRENT_SOURCE_DIR}/tests/*.cc
-    WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
-    COMMENT "Running clang-format"
-    VERBATIM)
-add_dependencies(qa format-tools)
+    add_custom_target(format-tests
+        COMMAND ${CLANG_FORMAT} -i -style=file ${ALL_TEST_SRCS}
```

Comment:
I've put a separate clang-format invocation in src/CMakeLists.txt, too.  This means we call it twice, but it doesn't have to reference files between the two directories.  Does that make sense?